### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by princerajputana6**
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+This is a opwn source project all of you can get benefit from it.
```
Comment (nytai) [inline README.md:1]: This implied and covered in the license section

**Comment by nytai** [comment]
closing this as there is no description and title is very generic

**Comment by ktmud** [comment]
Seems like a [Hacktoberfest](https://hacktoberfest.digitalocean.com/) spam lol.

**Comment by nytai** [comment]
yup, looks like they opened this same pr across 10 different repos

**Pull request closed without merging**